ios/FluentUI/Navigation/NavigationBarTokenSet.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Design token set for the `NavigationBar` control.
+public class NavigationBarTokenSet: ControlTokenSet<NavigationBarTokenSet.Tokens> {
+    public enum Tokens: TokenSetKey {
+        /// Describes the background color for the navigation bar.
+        case backgroundColor
+
+        /// Describes the color of the buttons at the top of the navigation bar.
+        case buttonTintColor
+
+        /// Describes the font used for a "large" title.
+        case largeTitleFont
+
+        /// Describes the color of the subtitle.
+        case subtitleColor
+
+        /// Describes the font used for the subtitle.
+        case subtitleFont
+
+        /// Describes the color of the title.
+        case titleColor
+
+        /// Describes the font used for a "small" title.
+        case titleFont
+    }
+
+    init(style: @escaping () -> NavigationBar.Style) {
+        super.init { [style] token, theme in
+            switch token {
+            case .backgroundColor:
+                return .uiColor {
+                    switch style() {
+                    case .primary, .default, .custom:
+                        return UIColor(light: theme.color(.brandBackground1).light,
+                                       dark: theme.color(.background3).dark)
+                    case .system:
+                        return theme.color(.background3)
+                    }
+                }
+            case .buttonTintColor, .subtitleColor: // By default, these return the same color
```

Comment:
Nit: I don't think this comment is necessary.

Reply:
I don't think it hurts to keep it.